packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/selectors.ts

```diff
@@ -19,10 +18,9 @@ import {
 } from '../exchange/services'
 import { getInputFromPair, getOutputFromPair } from '../swap/model'
 import { getRate } from '../swap/utils'
+import { LIMIT } from './model'
 import { SBCardStateEnum, SBCheckoutFormValuesType } from './types'
 
-const SDD_LIMIT = { min: '500', max: '10000' } as SDDLimits
-
```

Comment:
cool 👏

Reply:
I moved several consts to the model file in the simple buy data dir to clean up the template files a bit.